LoopTests/ViewModels/BolusEntryViewModelTests.swift

```diff
@@ -158,6 +158,7 @@ class BolusEntryViewModelTests: XCTestCase {
         bolusEntryViewModel.enteredManualGlucose = Self.exampleManualGlucoseQuantity
         XCTAssertEqual(Self.exampleBolusQuantity, bolusEntryViewModel.enteredBolus)
         waitOnMain()
+        waitOnMain()
```

Comment:
Why two calls to `waitOnMain`? Why not just use `waitOnMain(timeout: 2)`?

Reply:
Also applies to all other cases where `waitOnMain` is called twice in a row.

Reply:
This is the biggest mystery to me.  I have tried increasing the `timeout`, but that doesn't help.  There's something about having to wait twice that is required to make these tests pass.  My other thought was that some other test isn't cleaning up properly, leaving some extra cruft in the `.main` queue, but that doesn't seem to be true, either.  What I will say is that `BolusEntryViewModel` plays some weird tricks with `Combine`, including `debounce`-ing (which I _thought_ was disabled for unit tests).  I could put a comment in here, but I have been unable to get to the bottom of it 🤷‍♂️ .  Any ideas?

Reply:
Nope, you've tried what I would have tried. Sounds good.